src/server/src/controllers/comment.controllers.ts

```diff
@@ -0,0 +1,108 @@
+import { Request, Response } from 'express';
+import Author from '../models/Author';
+import Comment from '../models/Comment';
+import Post from '../models/Post';
+import { AuthenticatedRequest } from '../types/auth';
+import { PaginationRequest } from '../types/pagination';
+
+const createComment = async (req: AuthenticatedRequest, res: Response) => {
+  if (req.params.comment_id) {
```

Comment:
This condition will never be satisfied. There is no route where the `comment_id` will ever exists. Not in routes or requirements

Reply:
Yep, you are correct. I was using the post controller as a template for the comment controller initially. I will remove that condition.